dispatcher/src/main/kotlin/pl/droidsonroids/testing/mockwebserver/YamlResourcesParser.kt

```diff
@@ -1,14 +1,17 @@
 package pl.droidsonroids.testing.mockwebserver
 
+import org.apache.commons.text.translate.JavaUnicodeEscaper
 import org.yaml.snakeyaml.Yaml
 
 internal class YamlResourcesParser : ResourcesParser {
+    private val escaper = JavaUnicodeEscaper.above(0xD800)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.droidsonroids.pl/coding_rules#rule_key=detekt-kotlin%3AMagicNumber)